packages/visualizations/src/components/Map/Combinations/Tooltip.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+    export let label = '';
+    let containerElement: HTMLDivElement;
+    let x: number;
+    let y: number;
+    let hovered = false;
+
+    function showTooltip() {
+        hovered = true;
+        x = containerElement.offsetLeft;
+        y = containerElement.offsetTop + containerElement.offsetHeight + 2;
+    }
+
+    function hideTooltip() {
+        hovered = false;
+    }
+</script>
+
+<div
+    class="tooltip-container"
+    bind:this={containerElement}
+    on:mouseover={showTooltip}
```

Comment:
I don't know if we ever discussed mobile here with the Product Design team. The tooltip isn't displayed (although clicking repeatedly in Firefox's mobile testing seems to sometimes trigger it), and I'm not sure what we want to do here. Otherwise, mobile users will never have a label to describe the buttons...

Reply:
I think there is a better event to be used `on:touch` or something like that, that cover both hover and touch. I'll dig.